README.md

```diff
@@ -12,6 +12,8 @@
 
 - `token`: **Required** - Telegram bot's authorization token. Use GitHub secrets.
 - `to`: **Required** - Unique identifier or username of the target Telegram chat.
+- `thread-id`: Optional - Identifier of a topic in a Telegram supergroup.
+    - The message would not be sent if `to` is not a supergroup and `thread-id` is set.
```

Comment:
```suggestion
- `thread-id`: (Optional) Topic identifier in a Telegram supergroup.  
    - The message won’t be sent if `to` isn’t a supergroup and `thread-id` is set.
```

Reply:
Same here, is topic different from thread? We should stay consistent with the naming to not cause confusion.